server/graphql/v2/query/CryptoExchangeRateQuery.js

```diff
@@ -0,0 +1,22 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import { getCryptoFxRate } from '../../../lib/currency';
+
+const CryptoExchangeRateQuery = {
+  type: GraphQLString,
+  args: {
+    cryptoCurrency: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Crypto currency symbol. Example: BTC',
+    },
+    collectiveCurrency: {
+      type: new GraphQLNonNull(GraphQLString),
```

Comment:
Generally, a currency should always be an Enum.

Reply:
I've changed both the `collectiveCurrency` as well as the `cryptoCurrency` to enums.